src/reputation/tests/test_upvote_rsc_distribution.py

```diff
@@ -219,63 +127,5 @@ def test_ineligible_enhanced_distribution(self):
         eligible_user.reputation = 20000
         eligible_user.save()
 
-        distribution_amount = calculate_rsc_per_upvote()
-        distribution = create_upvote_distribution(
-            1, self.original_paper, GrmVote.objects.first()
-        )
+        distribution = distributions.Distribution(1, 1, 1)
         self.assertEquals(distribution.amount, 1)
-
-    def test_author_claim_pot(
```

Comment:
The claim tests might be different than this feature.

Reply:
It seems like it's testing the escrow payout for escrow that used to be created in the create_upvote_distribution method. Since that method is no longer used and the escrow isn't created, this isn't really testing anything anymore.

Reply:
Synced with Kobe. I will remove as this is not really testing anything anymore.